install_ffmpeg.sh

```diff
@@ -187,7 +227,7 @@ if [[ ! -e "$ROOT/ffmpeg/libavcodec/libavcodec.a" ]]; then
     --extra-ldflags="${EXTRA_FFMPEG_LDFLAGS} -L${ROOT}/compiled/lib -L/usr/local/cuda/lib64" \
     --prefix="$ROOT/compiled" \
     $EXTRA_FFMPEG_FLAGS \
-    $DEV_FFMPEG_FLAGS
+    $DEV_FFMPEG_FLAGS || (tail -100 ${ROOT}/ffmpeg/ffbuild/config.log && exit 1)
```

Comment:
What is this `tail` here? Looks weird, could you at least explain it with a comment?

Reply:
I think the intention here is that if `configure` fails, then print the last 100 log lines for debugging and exit.

Reply:
Yeah that!
```suggestion
    $DEV_FFMPEG_FLAGS || (tail -100 ${ROOT}/ffmpeg/ffbuild/config.log && exit 1)
    # If configure fails, then print the last 100 log lines for debugging and exit.
```